app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -15,12 +16,40 @@ class RecentPatientsView(context: Context, attrs: AttributeSet) : RecyclerView(c
     val groupAdapter = GroupAdapter<com.xwray.groupie.ViewHolder>()
     adapter = groupAdapter
 
-    groupAdapter.add(RecentPatientItem())
-    groupAdapter.add(RecentPatientItem())
-    groupAdapter.add(RecentPatientItem())
-    groupAdapter.add(RecentPatientItem())
-    groupAdapter.add(RecentPatientItem())
-    groupAdapter.add(RecentPatientItem())
-    groupAdapter.add(RecentPatientItem())
+    groupAdapter.add(RecentPatientItem(RecentPatientItem.Data(
```

Comment:
Please move adding this data for testing to a separate method.
You can also add a comment to it saying that this is only for testing until the controller is ready.